x/gov/types/keys.go

```diff
@@ -37,10 +37,11 @@ const (
 //
 // - 0x30: Params
 var (
-	ProposalsKeyPrefix          = []byte{0x00}
-	ActiveProposalQueuePrefix   = []byte{0x01}
-	InactiveProposalQueuePrefix = []byte{0x02}
-	ProposalIDKey               = []byte{0x03}
+	ProposalsKeyPrefix            = []byte{0x00}
+	ActiveProposalQueuePrefix     = []byte{0x01}
+	InactiveProposalQueuePrefix   = []byte{0x02}
+	ProposalIDKey                 = []byte{0x03}
+	VotingPeriodProposalKeyPrefix = []byte{0x04}
```

Comment:
Can you update the spec/README.md file too? Just add some info about this new state

Reply:
Doing it now, ty! As a note I think the store section is not entirely accurate (or at least not very easy to understand), but I'll that there and re-visit later.